src/ass/to_ass.py

```diff
@@ -332,14 +63,22 @@ def to_events(
         to_title_event(song, shouldPrintTitle),
         to_divider_event(song, "Romaji"),
         *[
-            to_romaji_event(
-                line, song.title, actorToStyle, switchDuration, transitionDuration
+            to_default_event(
+                preproc_line_text(line, i + 1),
+                actorToStyle,
+                switchDuration,
+                transitionDuration,
             )
-            for line in song.lyrics
+            for i, line in enumerate(song.lyrics)
         ],
         to_divider_event(song, "English"),
         *[
-            to_en_event(line, actorToStyle, switchDuration, transitionDuration)
-            for line in song.lyrics
+            to_default_event(
+                preproc_line_text_en(line, i + 1),
```

Comment:
Suggestion: `preproc_line_text` -> `to_romaji_k_line`

Since the mental model here seems to be SongLine -> KLine -> Event, and there are two different types of transformers, one that takes in a SongLine to generate a KLine and one that takes in a KLine and generates an Event

Reply:
Also copied the function name from `karaskel.lua` but sure, that sounds better.